generator/aggregating_sponsored_dataset.cpp

```diff
@@ -0,0 +1,25 @@
+#include "generator/aggregating_sponsored_dataset.hpp"
+
+namespace generator
+{
+SponsoredDataset::ObjectId AggregatingSponsoredDataset::FindMatchingObjectId(FeatureBuilder1 const & fb) const
+{
+  // There is only one source for now.
+  return m_datasets[0]->FindMatchingObjectId(fb);
+}
+
+size_t AggregatingSponsoredDataset::Size() const
+{
+  return accumulate(begin(m_datasets), end(m_datasets), size_t{},
```

Comment:
As there is only one source for now, why not to directly return m_datasets[0]->Size()?